<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QustionMathUi</class>
 <widget class="QWidget" name="QustionMathUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QustionMathUi</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5,2">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>155</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>272</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_10" native="true">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
font-family:Consolas,'楷体';
font-size:20px;
</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QtMaterialCheckable" name="CB_opt_addition">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>加法运算</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>最大值：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QtMaterialTextField" name="LE_addition_max">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QtMaterialCheckable" name="CB_opt_subtraction">
           <property name="text">
            <string>减法运算</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QtMaterialCheckable" name="CB_subtraction_hasNegative">
           <property name="text">
            <string>结果包含负数</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>最大值：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QtMaterialTextField" name="LE_subtraction_max">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QtMaterialCheckable" name="CB_opt_multiplication">
           <property name="text">
            <string>乘法运算</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QWidget" name="widget_8" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>最大值：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QtMaterialTextField" name="LE_multiplication_max">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QtMaterialCheckable" name="CB_opt_division">
           <property name="text">
            <string>除法运算</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QtMaterialCheckable" name="CB_division_thanZero">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>结果大于0</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QtMaterialCheckable" name="CB_division_hasRemainder">
           <property name="text">
            <string>结果包含余数</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QWidget" name="widget_12" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>结果保留几位小数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="SB_division_number_decimal">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>最大值：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QtMaterialTextField" name="LE_division_max">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialRaisedButton" name="btn_start">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>开  始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialRaisedButton" name="btn_return">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>返  回</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>271</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>155</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialtextfield.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialCheckable</class>
   <extends>QCheckBox</extends>
   <header location="global">lib/qtmaterialcheckable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MathTopic.qrc"/>
 </resources>
 <connections/>
</ui>
